src/frontend/packages/core/src/core/entity-catalogue/entity-catalogue-entity.ts

```diff
@@ -56,6 +57,8 @@ export class StratosBaseCatalogueEntity<T extends IEntityMetadata = IEntityMetad
    * If no schemaKey is provided then the default schema will be returned
    */
   public getSchema(schemaKey?: string) {
+    // TODO: schemaKey - ensure wherever this is called it contains the correct schemaKey (with respect to any config
+    // EntityCatalogueEntityConfig that may use a schemeKey different than that provided by entityCatalogue.getEntity's)
```

Comment:
Can you elaborate?

Reply:
We need to run through usages and check that the correct schemaKey is used. Think is most cases this should come from an `EntityCatalogueEntityConfig` object (whether the schema key is populated or not). We then need to ensure that this has the correct schema key when different from the default one (for service instances schemas with/without space, everything related to this issue is marked with a `// TODO: schemaKey`).